### PR TITLE
added function to coerce user keys into strings, added some tests

### PR DESCRIPTION
**Pull request by pkaeding**
See https://github.com/launchdarkly/python-client/pull/41/files for the moral equivalent in python.

### CODE REVIEW

**Comment by jkodumal** [comment]
Tests failed.

**Comment by apucacao** [comment]
yeah we're looking into it

File: index.js [hunk 1]
```diff
@@ -463,3 +466,9 @@ function intersect_safe(a, b)
     return b.indexOf(value) > -1;
   });
 }
+
+function sanitize_user(u) {
+  if (u['key']) {
+    u['key'] = "" + u['key'];
```
Comment (apucacao) [inline index.js]: minor, but I'm curious why you did not use `Object.prototype.toString()`.

**Comment by apucacao** [comment]
lgtm

**Pull request merged**